Content.Shared/Zombies/ZombieComponent.cs

```diff
@@ -2,37 +2,65 @@
 using Content.Shared.Chemistry.Reagent;
 using Content.Shared.Damage;
 using Content.Shared.Roles;
-using Content.Shared.Humanoid;
 using Content.Shared.StatusIcon;
 using Robust.Shared.Audio;
 using Robust.Shared.GameStates;
 using Robust.Shared.Prototypes;
-using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom;
 using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
-using static Content.Shared.Humanoid.HumanoidAppearanceState;
 
 namespace Content.Shared.Zombies;
 
-[RegisterComponent, NetworkedComponent]
+/// <summary>
+///   Applied to any mob that is carrying a zombie infection in any state.
+///   That mob will also have one of these additional components depending on the stage of infection:
+///   - InitialInfectedComponent - Human Patient0 players who are not being hurt by the infection yet.
+///   - PendingZombieComponent - The painful stage of the infection.
+///   - LivingZombieComponent - An active zombie either alive or in crit (and about to heal back to life)
+///   - [none of these] - This zombie has turned from undead to actually dead.
+/// </summary>
+[RegisterComponent, NetworkedComponent, Access(typeof(SharedZombieSystem))]
 public sealed partial class ZombieComponent : Component
 {
-    /// <summary>
+    public void CopyFrom(ZombieComponent other)
+    {
+        // Victims copy their settings from the zombie that bit them
+        MinInfectionChance = other.MinInfectionChance;
+        MaxInfectionChance = other.MaxInfectionChance;
+        InfectionTurnTime = other.InfectionTurnTime;
+        DeadMinTurnTime = other.DeadMinTurnTime;
+        PassiveHealing = other.PassiveHealing;
+        VirusDamage = other.VirusDamage;
+        AttackDamage = other.AttackDamage;
+        HealingOnBite = other.HealingOnBite;
+    }
+
```

Comment:
SerializationManager lets you copy components, you don't need to implement it yourself.